### ISSUE TITLE
Tiny changes on toolbar

### ISSUE THREAD

**Issue by Nikks84** [fast4x/RiMusic#955]
### Description

![Screenshot_2024-03-07-15-28-49-93](https://github.com/fast4x/RiMusic/assets/160481076/a79559f9-3b74-414f-bd11-793cf2de0d24)
1. As we see the ss the main player toolbar both side corner he looks simple It could be better than that. 
![Screenshot_2024-03-07-15-30-16-25](https://github.com/fast4x/RiMusic/assets/160481076/a4edd5c6-1c5e-477f-b541-59f6b186c5fc)
2.  Same on queue toolbar as well

### Suggested Solution

It would be better to little bit rounded  corners both main player toolbar & queue toolbar. I understand it will look even better than before.
Ex: 
![IMG_20240307_154159](https://github.com/fast4x/RiMusic/assets/160481076/4f8af649-4924-494c-84f4-90c1955be468)
Like the above ☝️ ss
 I remember a little bit that someone requested to toolbar changes about this if still could be implement these tiny changes enhanced the user experience it would be better the app ui.
 Other than app ui already attractive & rich. Thanx

**Comment by fast4x** [comment]
Yes, but i have not been able to round the corners in the bar, if I can I will do it later because it is not priority.

**Comment by Nikks84** [comment]
> Yes, but i have not been able to round the corners in the bar, if I can I will do it later because it is not priority.

Ohh.. 
It's ok no problem, but in future you decide you can be no problem.
